docs/content/en/docs/packages/cluster-autoscaler/addclauto.md

```diff
@@ -15,27 +15,13 @@ Be sure to refer to the [troubleshooting guide]({{< relref "../troubleshoot" >}}
    * While following this guide to install packages on a workload cluster, please make sure the `kubeconfig` is pointing to the management cluster that was used to create the workload cluster. The only exception is the `kubectl create namespace` command below, which should be run with `kubeconfig` pointing to the workload cluster.
```

Comment:
I thought we aren't installing packages on a workload cluster?

Reply:
The package itself gets installed to the workload cluster - I think that's what this references.